eip.tf

```diff
@@ -0,0 +1,29 @@
+resource "aws_eip" "this" {
```

Comment:
I would add a variable "manage_eip" and make EIP optional (disabled by default).

LMK what you think

Reply:
This is some change that should be discussed before sprint started. I could create additional task to do this.
@AutomationD

Reply:
You are right. Sorry, I missed that this is a move also, I thought you're adding EIP inernally. My bad.